include/native/windows/windows_base.h

```diff
@@ -31,6 +31,9 @@ typedef wchar_t WCHAR;
 typedef WCHAR *NWPSTR, *LPWSTR, *PWSTR;
 typedef unsigned char UCHAR, *PUCHAR;
 
+typedef char CHAR;
+typedef const CHAR *PCSTR;
```

Comment:
do we need LPCSTR or is that already defined somewhere?

Reply:
I've added it :)